app/src/main/java/org/sopt/and/component/CustomTextField.kt

```diff
@@ -0,0 +1,58 @@
+package org.sopt.and.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import org.sopt.and.R
+import org.sopt.and.ui.theme.WavveTheme
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.VisualTransformation
+import java.lang.Error
+
+@Composable
+fun AuthTextField(
+    modifier: Modifier,
+    value: String,
+    onValueChange: (String) -> Unit,
+    placeholder: String,
+    suffix: @Composable() (() -> Unit)? = null,
+    keyboardOptions: KeyboardOptions,
+    visualTransformation: VisualTransformation = VisualTransformation.None,
+    isError: Boolean = false
```

Comment:
안드로이드(컴포즈) 코딩 컨벤션에 따르면,

필수 파라미터는 선택 파라미터 위에 있어야 하고, Modifier는 모든 선택 파라미터 중 가장 상위에 있어야합니다.
```suggestion
fun AuthTextField(
    value: String,
    onValueChange: (String) -> Unit,
    placeholder: String,
    keyboardOptions: KeyboardOptions,
    modifier: Modifier = Modifier,
    suffix: @Composable() (() -> Unit)? = null,
    visualTransformation: VisualTransformation = VisualTransformation.None,
    isError: Boolean = false
```